packages/data-models/appConfig.ts

```diff
@@ -106,8 +106,16 @@ const APP_ROUTE_DEFAULTS = {
   ],
 };
 
+type IHeaderColourOptions = "primary" | "secondary" | "none";
+
 const APP_HEADER_DEFAULTS = {
   title: "App",
+  collapsing: false,
+  colour: "primary" as IHeaderColourOptions,
+  compact: false,
+  // expose heights (in px) so that "compact" can be applied by config/skin independent of theme
+  heightDefault: 56,
+  heightCompact: 36,
```

Comment:
nit(non-blocking) do we want authors to be able to manipulate these heights? I'm assuming there is reason for the numbers (e.g. minimum padding), so might be better to just expose the `compact` property and use that to manage heights from the scss

Reply:
I've updated `APP_HEADER_DEFAULTS` to include a `heightsMap` instead (see latest code changes):
```js
  heightsMap: {
    compact: 36,
    default: 56,
  }
```

The motivation for including these values in the appConfig was less about exposing them to authors but more to make available universally to the code. We could put this heightsMap object somewhere else and import it statically into the relevant places (header component and collapse-on-scroll directive). 

Or, we could set these values in the theme, i.e. as global CSS variables like `--header-height-default` and `--header-height-compact`. In that case we'd need a way to access them in the TS. Perhaps it would be sensible for the theme service to expose an observable for subscription that contains all of the custom CSS properties (starting with "--"). We already have a method in the theme service, `listAllCssCustomProperties()`. But would these additional subscriptions be worth it to just access these properties?